opendatahub-io/notebooks
# Upgrade odh-elyra 3.16.7 for run_url update

Upgrade odh-elyra 3.16.7 for run_url update
https://github.com/opendatahub-io/elyra/releases/tag/3.16.7

## Description


Set the run_url with experiment info included

The run_url is set in the format of: `{experiment_id}/runs/{run_id}`
Example:
```
https://rhods-dashboard-redhat-ods-applications.apps.datahub.redhat.com/experiments/experiment/024e260b-75e9-407e-9e8d-3974320c06dc/runs/b6c920a2-a3de-452c-bf58-88d46f3da65b
```

Related-to: https://issues.redhat.com/browse/RHOAIENG-8398

## How Has This Been Tested?




This PR needs to be run with the compliment of the Dashboard Image:
Use the following image: `quay.io/opendatahub/odh-dashboard:main-a8d2118`

1. Set up a DS project
2. Create a Data connection
3. Create Pipeline configuration
4. Use data-science notebook image create by this PR.
5. Execute a Pipeline
and check the RUN Details pointing to right pipeline run executed
![Screenshot from 2024-06-21 05-10-17](https://github.com/opendatahub-io/notebooks/assets/14028058/1937a6d3-2844-4049-b2b6-71ad1f083f61)


Expected:
URL should be of the form: `https://rhods-dashboard-redhat-ods-applications.apps.datahub.redhat.com/experiments/experiment/024e260b-75e9-407e-9e8d-3974320c06dc/runs/b6c920a2-a3de-452c-bf58-88d46f3da65b`

## Merge criteria:
